### PR TITLE
Don't start Decode on monitor thread if decoding is disabled

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by aleksbykov** [comment]
@bentsi @fruch This fix for decoding backtraces on monitoring node. If decode_bactrace is false or not set, don't start the decoding thread
Before sct.log had a lot of error messages that queue was not created

File: sdcm/cluster.py [hunk 1]
```diff
@@ -922,7 +922,7 @@ def start_task_threads(self):
             self.start_journal_thread()
             self.start_backtrace_thread()
             self.start_db_log_reader_thread()
-        if 'monitor' in self.name:
+        if 'monitor' in self.name and Setup.DECODING_QUEUE:
```
Comment (bentsi) [inline sdcm/cluster.py]: there is an SCT param `backtrace_decoding` that controls if it is enabled

Comment (bentsi) [reply sdcm/cluster.py]: use `Setup.BACKTRACE_DECODING`

Comment (aleksbykov) [reply sdcm/cluster.py]: @bentsi i thought about it, but but use DECODING_QUEUE to be sure that queue thread is started

Comment (bentsi) [reply sdcm/cluster.py]: if the thread starts and `Setup.BACKTRACE_DECODING` is `false` we have a bug that we should know about.
(the thread should start only when we want it to, and the way to control it is by setting `backtrace_decoding` flag and nothing else)

**Comment by bentsi** [comment]
fixed in #1638

**Pull request closed without merging**